{{cookiecutter.profile_name}}/slurm-submit.py

```diff
@@ -58,4 +79,8 @@
     slurm_utils.ensure_dirs_exist(sbatch_options[o]) if o in sbatch_options else None
 
 # submit job and echo id back to Snakemake (must be the only stdout)
-print(slurm_utils.submit_job(jobscript, **sbatch_options))
+jobid = slurm_utils.submit_job(jobscript, **sbatch_options)
+logger.debug("Registering %s with sidecar...", jobid)
+register_with_sidecar(jobid)
```

Comment:
Wouldn't this imply that --cluster-sidecar needs to be set?

Reply:
Yes, `config.yaml` contains the line

```yaml
cluster-sidecar: "slurm-sidecar.py
```